Library/Source/NapiBridge.h

```diff
@@ -82,61 +82,61 @@ namespace babylon
-                .AddVoidReturningMethod<&NativeEngineT::SetMatrix>("setMatrix")
-                .AddVoidReturningMethod<&NativeEngineT::SetIntArray>("setIntArray")
-                .AddVoidReturningMethod<&NativeEngineT::SetIntArray2>("setIntArray2")
-                .AddVoidReturningMethod<&NativeEngineT::SetIntArray3>("setIntArray3")
-                .AddVoidReturningMethod<&NativeEngineT::SetIntArray4>("setIntArray4")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloatArray>("setFloatArray")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloatArray2>("setFloatArray2")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloatArray3>("setFloatArray3")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloatArray4>("setFloatArray4")
-                .AddVoidReturningMethod<&NativeEngineT::SetMatrices>("setMatrices")
-                .AddVoidReturningMethod<&NativeEngineT::SetMatrix3x3>("setMatrix3x3")
-                .AddVoidReturningMethod<&NativeEngineT::SetMatrix2x2>("setMatrix2x2")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloat>("setFloat")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloat2>("setFloat2")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloat3>("setFloat3")
-                .AddVoidReturningMethod<&NativeEngineT::SetFloat4>("setFloat4")
-                .AddVoidReturningMethod<&NativeEngineT::SetBool>("setBool")
-                .AddValueReturningMethod<&NativeEngineT::CreateTexture>("createTexture")
-                .AddVoidReturningMethod<&NativeEngineT::LoadTexture>("loadTexture")
-                .AddVoidReturningMethod<&NativeEngineT::LoadCubeTexture>("loadCubeTexture")
-                .AddValueReturningMethod<&NativeEngineT::GetTextureWidth>("getTextureWidth")
-                .AddValueReturningMethod<&NativeEngineT::GetTextureHeight>("getTextureHeight")
-                .AddVoidReturningMethod<&NativeEngineT::SetTextureSampling>("setTextureSampling")
-                .AddVoidReturningMethod<&NativeEngineT::SetTextureWrapMode>("setTextureWrapMode")
-                .AddVoidReturningMethod<&NativeEngineT::SetTextureAnisotropicLevel>("setTextureAnisotropicLevel")
-                .AddVoidReturningMethod<&NativeEngineT::SetTexture>("setTexture")
-                .AddVoidReturningMethod<&NativeEngineT::DeleteTexture>("deleteTexture")
-                .AddVoidReturningMethod<&NativeEngineT::DrawIndexed>("drawIndexed")
-                .AddVoidReturningMethod<&NativeEngineT::Draw>("draw")
-                .AddVoidReturningMethod<&NativeEngineT::Clear>("clear")
-                .AddValueReturningMethod<&NativeEngineT::GetRenderWidth>("getRenderWidth")
-                .AddValueReturningMethod<&NativeEngineT::GetRenderHeight>("getRenderHeight")
-                .Finalize();
+                .NapiBridge<NativeEngineT>::AddVoidReturningMethod<&NativeEngineT::RequestAnimationFrame>("requestAnimationFrame")
```

Comment:
This seems very weird, unless I'm reading this wrong. NapiBridge doesn't _have_ an AddVoidReturningMethod() method, does it? NapiBridge<T>::Definition has that method, not the parent class. Unless I've missed something...